exercises/02-generate-odata-clients.md

```diff
@@ -0,0 +1,28 @@
+# Generate your own OData clients
+
+There are many scenarios where you have either your own OData service or want to integrate with another SAP OData service. For those cases, you can generate your own OData client as described in the following.
+
+## Download EDMX file
+
+Download the service specification file for [SAP S/4HANA Cloud Workforce Timesheet service](https://api.sap.com/api/API_MANAGE_WORKFORCE_TIMESHEET/overview) and [SAP SuccessFactors Employee Central Time Off service](https://api.sap.com/api/ECTimeOff/overview) from the SAP Business API Hub and put it into the [resources/service-specs](../resources/service-specs/).
+
+In this directory, you will also find the [service mapping file](../resources/service-mapping.json), which contains some extra configuration for the generated service(s), like the directory name and service path for the service.
+
+## Run the generation
+To generate the OData clients, just run an npm script:
+```sh
+npm run generate
+```
+Once this is completed you should find some generated code, your OData clients, in the [src/generated](../src/generated/). We will use this code for OData requests to SAP S/4HANA and SAP SuccessFactors.
+>### Under the food: What does the script actually do?
```

Comment:
[q] what does this `>` formatting do? I think a normal subsection is fine.

Reply:
I converted this quote section into subsection.